### PR TITLE
Add gradle.properties to the gradle plugin project

### LINKED ISSUE #3804: Gradle plugin module is missing gradle.properties
Gradle build scripts are referencing `${version}` property which is passed in as a system property when the Gradle build process is launched from the Maven build.
However when the project is imported into an IDE as a Gradle project, this variable cannot be resolved as it's not defined in the "Gradle space". And also for the same reason `gradle build` fails.
An easy fix would be to add `gradle.properties` file containing the current project version.

### PR DESCRIPTION
**Pull request by aloubyansky**
Fixes #3804

### CODE REVIEW

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
@gsmet mentioned this is OK for the release process [here](https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/gradle.20buildDev)

**Comment by gsmet** [comment]
Please wait for me to update the release process before merging this one.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
I made the changes to the release process.

**Pull request merged**
